server/model/work.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+export type Work = {|
+  id: string;
+  label: string;
+  descripition: string;
```

Comment:
I should have been clearer: the word "descripition" is misspelt. Should be "description".